Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM mesosphere/dcos-system-test-driver:latest
 
 # Specify the component versions to use
-ENV NODE_VERSION="8.9.4" \
-  NPM_VERSION="5.7.1" \
+ENV NODE_VERSION="10.15.2" \
```

Comment:
I think we should also tell tools infra to update their dcos-ui image once this is merged 😇

Reply:
we cant, `dcos-ui` label is used by 1.10 branches